tests/conftest.py

```diff
@@ -138,7 +138,10 @@ def _get_images() -> dict[str, Union[SpatialImage, MultiscaleSpatialImage]]:
     dims_3d = ("z", "y", "x", "c")
     out["image2d"] = Image2DModel.parse(RNG.normal(size=(3, 64, 64)), dims=dims_2d, c_coords=["r", "g", "b"])
     out["image2d_multiscale"] = Image2DModel.parse(
-        RNG.normal(size=(3, 64, 64)), scale_factors=[2, 2], dims=dims_2d, c_coords=["r", "g", "b"]
+        RNG.normal(size=(3, 64, 64)),
```

Comment:
Here rgba would not test extra things. There are other places in the codebase where this would make a difference though: in napari-spatialdata and spatialdata-plot. I suggest to keep it the test as it is now and to test the rgba when we will merge the relative PRs from Andreas (this one https://github.com/scverse/napari-spatialdata/pull/153 and the ones that are linked in the PR).